src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,14 @@
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        boolean result = false;
+        UserService userService = new UserService();
+        if (userService.findByEmail(email) != null) {
+            if ((userService.findByEmail(email).getEmail() == email)
+                    & (userService.findByEmail(email).getPassword() == password)) {
+                result = true;
+            }
+        }
+        return result;
```

Comment:
у нас в двух разных `if` один `return`, их можем объединить, но первое условие во втором `if` нам не нужно (если метод `login` работает правильно), также ты много раз пишешь `userService.findByEmail(email)`, давай вынесем это в отдельную переменную, и напоследок мы можем сразу делать `return` вместо `result = true`, тогда наш код можно записать примерно так:
```suggestion
        UserService userService = new UserService();
        User user = userService.findByEmail(email);
        return user != null && user.getPassword() == password);
```
И еще, лучше использовать "&&", a не "&", тогда, если левое условие будет `false`, правое не будет проверяться